deploy/crds-old/4.4.0_performanceprofiles_crd.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Do we need this manifest here, isn't it enough to have it in `deploy/olm-catalog/performance-addon-operator/4.4.0`?

Reply:
done